frontends/web/src/routes/device/bitbox02/components/password-entry/password-entry.tsx

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright 2021 Shift Crypto AG
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { h, RenderableProps} from 'preact';
+import PasswordGestureVideo from './assets/password-gestures.webm';
+import * as styles from './password-entry.css';
+
+export interface IPasswordEntryProps {}
+
+function isVideoPlaying(video: HTMLVideoElement): boolean {
+    return !!(video.currentTime > 0 && !video.paused && !video.ended && video.readyState > 2);
```

Comment:
why `!!`? should be `boolean` already by the looks of it

Reply:
`true`

(pun intended)

Reply:
fixed in https://github.com/digitalbitbox/bitbox-wallet-app/compare/b4c7cc93b336bf409c01e9c0ac8f233133f4daf7..de055f00a4132747c17abc1964a5ee53b743ef8a